app/tests/components/Form/RadioWidget.test.tsx

```diff
@@ -0,0 +1,10 @@
+import RadioWidget from '../../../lib/theme/widgets/RadioWidget';
+
+describe('RadioWidget', () => {
```

Comment:
@wenzowski @marcellmueller I would like to get your input on these 3 tests I framed out. Would you add/remove any? Thoughts on testing strategies?